docs/ja/index.md

```diff
@@ -1,66 +1,62 @@
 ---
-toc_priority: 3
-toc_title: "\u6982\u89B3"
+toc_priority: 0
+toc_title: "概要"
 ---
 
-# ClickHouseとは? {#clickhousetoha}
+# ClickHouseとは? {#what-is-clickhouse}
 
 ClickHouseは、クエリのオンライン分析処理（OLAP）用の列指向のデータベース管理システム（DBMS）です。
 
 「通常の」行指向のDBMSでは、データは次の順序で保存されます。
 
-| Row | WatchID     | JavaEnable | Title              | GoodEvent | EventTime           |
-|-----|-------------|------------|--------------------|-----------|---------------------|
-| \#0 | 89354350662 | 1          | Investor Relations | 1         | 2016-05-18 05:19:20 |
-| \#1 | 90329509958 | 0          | Contact us         | 1         | 2016-05-18 08:10:20 |
-| \#2 | 89953706054 | 1          | Mission            | 1         | 2016-05-18 07:38:00 |
-| \#N | …           | …          | …                  | …         | …                   |
+Row | WatchID | JavaEnable | Title | GoodEvent | EventTime
+--- | --- | --- | --- | --- | ---
+#0 | 89354350662 | 1 | Investor Relations | 1 | 2016-05-18 05:19:20
```

Comment:
@tac0x2a could you revert these changes to tables? It seems like they are the reason for the failed tests.

Reply:
@blinkov Thanks to review! I fixed it!